ADAL/src/request/ADAcquireTokenSilentHandler.m

```diff
@@ -150,25 +174,15 @@ - (void)acquireTokenByRefreshToken:(NSString *)refreshToken
         request_data[MSID_OAUTH2_CLAIMS] = claims;
     }
     
-    NSString *authority = _requestParams.cloudAuthority ? _requestParams.cloudAuthority : _requestParams.authority;
-
-    __auto_type adfsAuthority = [[MSIDADFSAuthority alloc] initWithURL:[NSURL URLWithString:authority] context:nil error:nil];
-
-    BOOL isADFSInstance = adfsAuthority != nil;
-
-    if (!isADFSInstance)
+    NSString *userId = (cacheItem.accountIdentifier.legacyAccountId ?: _requestParams.identifier.userId);
+    NSString *enrollmentId = [self enrollmentIDForHomeAccountID:cacheItem.accountIdentifier.homeAccountId legacyUserID:userId];
+    
+    if (![NSString msidIsStringNilOrBlank:enrollmentId])
```

Comment:
what is the reason of removing the adfs check here? :)

Reply:
It's part of the enrollmentIdForHomeAccountId method